docs/toml.md

```diff
@@ -110,7 +110,23 @@
 #     CertFile = "integration/fixtures/https/snitest.org.cert"
 #     KeyFile = "integration/fixtures/https/snitest.org.key"
 #
-
+# To enable basic auth on an entrypoint
+# with 2 user/pass: test:test and test2:test2
+# Passwords are encoded in MD5: you can use htpasswd to generate those ones
```

Comment:
Hum, `md5` ? Out of curiosity, wouldn't it be better to be able to provide your own ? like `test:md5:$apr1$…` or `test:sha1:…` ?